include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -1084,45 +1086,41 @@ struct __early_exit_find_or
 {
     _Pred __pred;
 
-    template <typename _NDItemId, typename _IterSize, typename _WgSize, typename _LocalFoundState, typename _BrickTag,
-              typename... _Ranges>
+    template <typename _NDItemId, typename _SrcDataSize, typename _DataPerWorkItemSize, typename _LocalFoundState,
```

Comment:
It looks like `_DataPerWorkItemSize` is outdated. Should we rename it to something like `_IterationDataSize` to match the parameter's name?

Reply:
Fixed